EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -201,9 +264,9 @@ protected void onUpdate(AjaxRequestTarget target) {
         layoutForm.add(listContainer);
         listContainer.setOutputMarkupId(true);
 
-        DropDownChoice<EapEventType> dropDown = new DropDownChoice<>("eventTypeField",
-                new PropertyModel<EapEventType>(this, "selectedEventType"), eventTypes);
-        dropDown.add(new AjaxFormComponentUpdatingBehavior("onchange") {
+        eventTypeDropDown = new DropDownChoice<>("eventTypeField",
+                new PropertyModel<EapEventType>( this, "selectedEventType" ), eventTypes);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') '(' is followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AParen_Pad_BP2016)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') ')' is preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AParen_Pad_BP2016)